Logger/FileLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string fPath;
+        StreamWriter sWriter;
```

Comment:
Specify access modifiers on members.
```suggestion
        private StreamWriter sWriter;
```

This field is also only used inside of the Log method and could have simply be a local member rather than field.